### ISSUE TITLE
Suggestion - instead of using classes to select the h2s, use IDS

### ISSUE THREAD

**Issue by jzhu01** [kryx7/prj-rev-bwfs-dasmoto#3]
Seen here: https://github.com/kryx7/prj-rev-bwfs-dasmoto/blob/master/DasMoto'sArts%26Crafts/resources/index.css#L11-L19

Since the classes are usually repeated, but the code for the h2 background color is unique to that element, it makes more sense to use IDs here. 

Likewise, the strong elements should have a class, instead an ID, because that is repeated.

**Comment by kryx7** [comment]
Yes, I was thinking the same, but I didn't manage to find an effective way of adding the IDs to h2. In your summary you already suggested to use the <strong> tag along with the IDs and that is what I will be doing in the future. Thank you!
